Formula/f/ffmpeg.rb

```diff
@@ -117,6 +118,7 @@ def install
       --enable-libjxl
       --enable-libmp3lame
       --enable-libopus
+      --enable-libplacebo
```

Comment:
Could you explain why we want to enable this, and how a significant number of current users of the FFmpeg formula would find it useful?

Reply:
> Could you explain why we want to enable this, and how a significant number of current users of the FFmpeg formula would find it useful?

[Released two years ago in FFmpeg 5.0](https://github.com/FFmpeg/FFmpeg/blob/746df0a19aab263bdba26d3ea9ca7e05e95b9db6/Changelog#L41), it allows libplacebo to be used as [a video filter](https://ffmpeg.org/ffmpeg-filters.html#libplacebo) during encoding. I would expect FFmpeg's filters to work out-of-the-box (like [frei0r does](https://github.com/Homebrew/homebrew-core/blob/fa95266536f90ddf2f9b89a96579f0c4923a1b50/Formula/f/ffmpeg.rb#L141)) without having to modify the formula to add additional compiler flags. (The filter was added to FFmpeg so that users could use it, right?) The kind of power-users who would use FFmpeg are certainly interested in the high performance and GPU-acceleration that libplacebo provides when most other filters are CPU-only. Personally, I opened this pull request after I got an error saying that libplacebo wasn't enabled when I tried to use its high-quality scaling functionality.

Reply:
This adds `libplacebo`, `shaderc`, `vulkan-loader` and `vulkan-headers` to `ffmpeg`'s already-bloated dep tree.

I'd rather have multiple users ask for this first before adding it to everyone's `ffmpeg` installs.

Reply:
Strongly agreed with @carlocab.